components/automate-deployment/pkg/backup/artifact_streams.go

```diff
@@ -437,6 +443,123 @@ func NewCountingStream(stream ArtifactStream) CountingStream {
 	}
 }
 
+// ReplayableStream is a stream that consumes a stream to a file and allows
+// it to be replayed several times
```

Comment:
"several" makes it sound like there is a limit to the number of times you can replay a stream. Is that the case?

Reply:
There's no limit, you just need to call `Reset()` in order to get a new stream.